plugin/al/lib/AL_USDMaya/AL/usdmaya/Global.h

```diff
@@ -16,10 +16,10 @@
 #pragma once
 
 #include "AL/usdmaya/Api.h"
-
-#include "AL/usd/utils/ForwardDeclares.h"
 #include "AL/event/EventHandler.h"
 
+#include <usdUtils/ForwardDeclares.h>
```

Comment:
Why are you forcing the compiler to search the -isystem paths first? Paths specified with -I should use "", not <>. This will just slow down the build.

Reply:
> By default, the preprocessor looks for header files included by the quote form of the directive #include "file" first relative to the directory of the current file, and then in a preconfigured list of standard system directories. For example, if /usr/include/sys/stat.h contains #include "types.h", GCC looks for types.h first in /usr/include/sys, then in its usual search path.

For the angle-bracket form #include <file>, the preprocessor’s default behavior is to look only in the standard system directories.

Given that this is a separate library, what exactly will be slowed down in this case? ForwardDeclares.h won't be in the same folder so preprocessor will go and look in the pre-configured list of directories.